packages/components/table/index.ts

```diff
@@ -37,3 +39,5 @@ export type {
   TableColumnSortable,
   TableColumnSortOrder,
 } from './src/types'
+
+export type IxTableColumn = typeof IxTableColumn
```

Comment:
```suggestion
export type TableColumnComponent = typeof IxTableColumn
```
记得把这个类型再 type.d.ts 里再声明一下